lib/errors/AlreadyExistsError.js

```diff
@@ -0,0 +1,11 @@
+var util = require('util');
+
+var AlreadyExists = function (message) {
```

Comment:
**Question:** The name does not affect the logic because it is exported, BUT could it make more sense to keep the "Error" in the name to keep the convention?
**Or** Do you think that it's unnecessary?